### DISCUSSION TITLE
Pin project(s) to profile.

### DISCUSSION THREAD

**Discussion by Akatmks** [community/community#34686] [Other Feature Feedback, Questions, & Ideas] [answered]
There is already a discussion #33631 about pinning projects to profile, but my reason for pinning projects is a little bit different so hope it would be fine for me to make a new discussion here.  

I have a public GitHub Project about my plans and schedules for future works. It would be great if I can pin it front and centre on my profile so when other community members visit my profile, they can get an idea where I'm heading towards. Thank you.

**Comment by jfullstackdev** [comment, marked as answer]
you can use external services that will add more than 6 pinned repos, I'm using it too

https://github.com/anuraghazra/github-readme-stats#github-extra-pins

**Reply by Akatmks** [reply]
I didn't know profile README exist! Thank you very much!
